docs/source/technical_tutorials/search/maap_stac_r.ipynb

```diff
@@ -0,0 +1,676 @@
+{
```

Comment:
<p>The following will cause an error if <code>length(items$features) == 0</code>:</p><p> </p>

<pre class="ql-syntax" spellcheck="false">first_item &lt;- items$features[[1]]
</pre><p> </p><p>Therefore, your previous length check was for naught.</p><p> </p><p>If you're going to include expressions that will fail on empty lists, then I suggest that you either <em>always</em> wrap such expressions within a length check, or <em>never</em> do so. In other words, don't mix the 2 approaches. Prefer consistency instead.</p><p> </p><p>In the context of this notebook, for simplicity and clarity, I would agree with Henry's suggestion to simply drop the length checks (previously null checks).</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/MAAP-Project/maap-documentation/pull/464/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>

Reply:
<p>Thanks, Chuck! Updated the code to account for the error. We went ahead and removed trycatch, but kept the length checks - the only reason being is because there is a collection with 0 items (AFRISAR_DLR2). However, if y'all prefer, we can still remove the length checks except for this updated cell (in case someone does use the empty collection I guess?)</p>